README.md

```diff
@@ -10,3 +10,6 @@ containers external IP, iptables will be configured to route container's traffic
 A chain named `EXTERNAL_IP` is created in the `nat` table into which all the rules are added.
 And one more empty chain is created after this one for any additional custom rules you might want
 to add, named `AFTER_EXTERNAL_IP`.
+
+Please make sure `/run/xtables.lock` exists on the host before starting the container.
```

Comment:
Can you explain why this mount is necessary?

Reply:
Added some extra explanation in the latest commit